src/foam/nanos/column/NestedPropertiesExpression.js

```diff
@@ -281,6 +281,9 @@ foam.CLASS({
         var exprArray = [];
         for ( var propName of propNames ) {
           var expr = objClass.getAxiomByName(propName) || foam.nanos.column.NestedPropertiesExpression.create({ objClass: objClass, nestedProperty: propName });
+          if ( foam.dao.OneToManyRelationshipProperty.isInstance(expr) ||
```

Comment:
This can be problematic, imagine things like referral codes, they have relationship to user and user is essential information on it

Reply:
user should be a reference property on referralcode so it should be fine, it just doesn't work for the 'many' side of the relationship, since they don't actually have a corresponding axiom on the java side